nestkernel/nestmodule.cpp

```diff
@@ -609,6 +609,17 @@ NestModule::CopyModel_l_l_DFunction::execute( SLIInterpreter* i ) const
   const std::string newmodname = getValue< std::string >( i->OStack.pick( 1 ) );
   DictionaryDatum dict = getValue< DictionaryDatum >( i->OStack.pick( 0 ) );
 
+  if ( oldmodname.toString() == "gap_junction" )
```

Comment:
This concerns the documentation above: The warning about module unloading should go, since we have removed module unloading (it never worked). Also, we should add a note saying that gap junctions cannot be copied.